### ISSUE TITLE
Window title cannot display unicode

### ISSUE THREAD

**Issue by Erxl** [Ultz/Silk.NET#282]
# Summary
Window title cannot display unicode

# Steps to reproduce
- Platform: Desktop
- Framework Version: .NET Core 3.1
- API: GLFW
- API Version: 

steps:
```csharp
var w = Window.Create(WindowOptions.Default);
w.Title = "测试";
w.Run();
```

**Comment by Perksey** [comment]
Related
https://github.com/glfw/glfw/issues/1199
https://github.com/glfw/glfw/blob/master/tests/title.c

**Comment by Perksey** [comment]
@HurricanKai This is possible now due to the new SilkMarshal surface, SilkTouch just needs to be adapted to allow LPUTF8Str. Assigning to you, but no rush on this as it's a niche use case.

**Comment by HurricanKai** [comment]
Resolved as part of #375. Added Lab test to verify.
![image](https://user-images.githubusercontent.com/22711887/102660627-a03bf980-417b-11eb-9ebf-7200edd0f06e.png)
